test/kwtool_web/features/registration_page/view_registration_page_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule KwtoolWeb.HomePage.ViewRegistrationPageTest do
+  use KwtoolWeb.FeatureCase, async: true
+
+  @path Routes.auth_path(KwtoolWeb.Endpoint, :show)
+
+  feature "view registration page", %{session: session} do
+    visit(session, @path)
+    assert_has(session, Query.text("Registration"))
+  end
+
+  feature "Registration", %{session: session} do
```

Comment:
Let's add more detail to this test description.

Reply:
Updated.